rules/windows/process_creation/proc_creation_win_proc_dump_dumpminitool.yml

```diff
@@ -16,14 +17,12 @@ logsource:
     product: windows
 detection:
     selection:
-        Image|endswith: '\DumpMinitool.exe'
-    selection_original_name:
-        OriginalName: 'DumpMinitool.exe'
-    selection_flags:
-        CommandLine|contains|all:
+        - Image|endswith: '\DumpMinitool.exe'
+        - OriginalFileName: 'DumpMinitool.exe'
+        - CommandLine|contains|all:
             - ' --processId '
             - ' --dumpType Full'
-    condition: 1 of selection*
+    condition: selection
```

Comment:
It is the same but I think this way it easier to understand the 2 detection approaches
```yaml
    selection_exe:
        - Image|endswith: '\DumpMinitool.exe'
        - OriginalFileName: 'DumpMinitool.exe'
    selection_flags:
        CommandLine|contains|all:
            - ' --processId '
            - ' --dumpType Full'
    condition: 1 of selection_*
```